Quick Paste/ViewController.swift

```diff
@@ -268,17 +282,19 @@ class ViewController: NSViewController {
 
 extension ViewController {
   // MARK: Storyboard instantiation
-  static func freshController() -> ViewController {
-    //1.
-    let storyboard = NSStoryboard(name: NSStoryboard.Name("Main"), bundle: nil)
-    //2.
-    let identifier = NSStoryboard.SceneIdentifier("ViewController")
-    //3.
-    guard let viewcontroller = storyboard.instantiateController(withIdentifier: identifier) as? ViewController else {
-      fatalError("Why cant i find ViewController? - Check Main.storyboard")
-    }
-    return viewcontroller
-  }
+//  static func freshController() -> ViewController {
+//    //1.
+//    let storyboard = NSStoryboard(name: NSStoryboard.Name("Main"), bundle: nil)
+//    print("inside viewcontroller freshcontroller")
+//    //2.
+//    let identifier = NSStoryboard.SceneIdentifier("ViewController")
+//    //3.
+//    guard let viewcontroller = storyboard.instantiateController(withIdentifier: identifier) as? ViewController else {
+//      fatalError("Why cant i find ViewController? - Check Main.storyboard")
+//    }
+//    print("viewcontroller:\(viewcontroller)")
+//    return viewcontroller
+//  }
```

Comment:
freshController shouldn't be used anymore, since when splitViewController, all children viewController will be instantiated, call freshController separately, end up duplicated controllers, which would lead to unexpected bare view.